SPECS/python-html5lib/python-html5lib.spec

```diff
@@ -59,14 +61,17 @@ sed -i 's/from mock import/from unittest.mock import/' html5lib/tests/test_meta.
 %pyproject_save_files html5lib
 
 %check
-pip3 install more-itertools umsgpack webencodings
+pip3 install more-itertools umsgpack webencodings iniconfig
```

Comment:
Could you also try replacing installing `more-itertools` through pip by adding `BuildRequires: python3-more-itertools`?

Reply:
Sure, new buddy build started here with changes https://dev.azure.com/mariner-org/mariner/_build/results?buildId=570210&view=results

Reply:
Done for both `pluggy` and `more-itertools` for all modified specs.